File: Lignes/Lignes.r
# Données fictives
x <- c("Jan", "Feb", "Mar", "Apr", "May")
y <- c(100, 110, 105, 115, 120)

# Création du graphique en lignes
plot(y, type = "o", col = "blue", main = "Graphique en Lignes",
     xlab = "Mois", ylab = "Prix", xaxt = "n", lwd = 2)
axis(1, at = 1:5, labels = x)  # Ajouter les étiquettes pour l'axe des x
grid()

File: Lignes/Lignes.py
import matplotlib.pyplot as plt

# Données fictives
x = ["Jan", "Feb", "Mar", "Apr", "May"]
y = [100, 110, 105, 115, 120]

# Création du graphique en lignes
plt.plot(x, y, marker='o', linestyle='-', color='blue', label="Prix mensuel")
plt.title("Graphique en Lignes")
plt.xlabel("Mois")
plt.ylabel("Prix")
plt.legend()
plt.grid(alpha=0.5)
plt.show()
